sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.updateProgramOffering.e2e-spec.ts

```diff
@@ -155,6 +160,181 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-updateProgramOffer
+        actualTuitionCosts: 1234,
+        programRelatedCosts: 3211,
+        mandatoryFees: 100001,
+        exceptionalExpenses: 555,
+      };
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .patch(endpoint)
+        .send(payload)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({});
+      const updatedEducationProgramOffering =
+        await db.educationProgramOffering.findOne({
+          where: { id: savedEducationProgramOffering.id },
+        });
+      expect(updatedEducationProgramOffering).toEqual({
+        createdAt: expect.any(Date),
+        updatedAt: expect.any(Date),
+        id: expect.any(Number),
+        name: payload.offeringName,
+        studyStartDate: payload.studyStartDate,
+        studyEndDate: payload.studyEndDate,
+        actualTuitionCosts: payload.actualTuitionCosts,
+        programRelatedCosts: payload.programRelatedCosts,
+        mandatoryFees: payload.mandatoryFees,
+        exceptionalExpenses: payload.exceptionalExpenses,
+        offeringDelivered: payload.offeringDelivered,
+        lacksStudyBreaks: payload.lacksStudyBreaks,
+        offeringType: payload.offeringType,
+        offeringIntensity: payload.offeringIntensity,
+        yearOfStudy: payload.yearOfStudy,
+        showYearOfStudy: payload.showYearOfStudy,
+        hasOfferingWILComponent: payload.hasOfferingWILComponent,
+        offeringWILType: null,
+        studyBreaks: {
+          totalDays: studyPeriodBreakdown.totalDays,
+          studyBreaks: [
+            {
+              breakStartDate: studyBreak.breakStartDate,
+              breakEndDate: studyBreak.breakEndDate,
+              breakDays: 32,
+              eligibleBreakDays: 21,
+              ineligibleBreakDays: 11,
+            },
+          ],
+          totalFundedWeeks: studyPeriodBreakdown.totalFundedWeeks,
+          fundedStudyPeriodDays: studyPeriodBreakdown.fundedStudyPeriodDays,
+          unfundedStudyPeriodDays: studyPeriodBreakdown.unfundedStudyPeriodDays,
+        },
+        offeringDeclaration: payload.offeringDeclaration,
+        assessedDate: null,
+        offeringStatus: OfferingStatus.CreationPending,
+        submittedDate: expect.any(Date),
+        courseLoad: expect.any(Number),
+      });
+    },
+  );
+
+  it(`Should return error when trying to update a new offering with exceptionalExpenses greater than ${MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE} is passed.`, async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    const fakeEducationProgram = createFakeEducationProgram({
+      institution: collegeF,
+      user: collegeFUser,
+    });
+    fakeEducationProgram.sabcCode = faker.random.alpha({ count: 4 });
+    const savedFakeEducationProgram = await db.educationProgram.save(
+      fakeEducationProgram,
+    );
+    const savedEducationProgramOffering =
```

Comment:
Just to double check, if the data is not inserted the error would be different and we really need the data setup to get this `BAD_REQUEST`, right?

Reply:
As mentioned, it is a `BAD_REQUEST` but with all the errors and warning, 
![image](https://github.com/bcgov/SIMS/assets/77353155/68429aff-3ad6-44e8-8535-683a68397a4f)